src/App.js

```diff
@@ -1,25 +1,182 @@
-import logo from './logo.svg';
 import './App.css';
+import firebase from './firebase.js';
+import { getDatabase, ref, onValue, update, get } from 'firebase/database';
+import { useEffect, useState } from 'react';
+
+import Header from './Header';
+import Footer from './Footer';
+
+const App = () => {
+  const [userChoice, setUserChoice] = useState(null)
+  const [computerChoice, setComputerChoice] = useState(null)
+  const [result, setResult] = useState(null)
+  const [userValue, setUserValue] = useState(0)
+  const [computerValue, setComputerValue] = useState(0)
+
+
+  const choices = ['rock', 'paper', 'scissors']
+
+  useEffect( () => {
+    const database = getDatabase(firebase)
+    const dbRef = ref(database)
+    onValue(dbRef, (response) => {
+      const data = response.val()
+      const {userScore, computerScore} = data
+      setUserValue(userScore)
+      setComputerValue(computerScore)
+    })
+  },[])
+
+  const increaseUserScore = () => {
+    const database = getDatabase(firebase)
+    const dbRef = ref(database)
+    get(dbRef).then((data) => { 
+      const oldData = data.val()
+      const newData = {
+        userScore: oldData.userScore +1,
+        drawScore: oldData.drawScore,
+        computerScore: oldData.computerScore
+      }
+      update(dbRef, newData)
+    }).catch((error) => {
+      console.log(error);
+    })
+  
+  }
+
+    const increaseComputerScore = () => {
+    const database = getDatabase(firebase)
+    const dbRef = ref(database)
+    get(dbRef).then((data) => { 
+      const oldData = data.val()
+      const newData = {
+        computerScore: oldData.computerScore +1,
+        drawScore: oldData.drawScore,
+        userScore: oldData.userScore
+      }
+      update(dbRef, newData)
+    }).catch((error) => {
+      console.log(error);
+    })
+  
+  }  
+  
+    const increaseDrawScore = () => {
+    const database = getDatabase(firebase)
+    const dbRef = ref(database)
+    get(dbRef).then((data) => { 
+      const oldData = data.val()
+      const newData = {
+        drawScore: oldData.drawScore +1,
+        userScore: oldData.userScore,
+        computerScore: oldData.computerScore
+      }
+      update(dbRef, newData)
+    }).catch((error) => {
+      console.log(error);
```

Comment:
Maybe adding an alert or displaying something to the page could help let the user know that something has gone wrong